cms/djangoapps/contentstore/video_storage_handlers.py

```diff
@@ -13,12 +13,11 @@
 import shutil
 import pathlib
 import zipfile
+import boto3
 
 from contextlib import closing
 from datetime import datetime, timedelta
 from uuid import uuid4
-from boto.s3.connection import S3Connection
-from boto import s3
```

Comment:
I think this is the last of the mentions of boto in our system, we should drop boto from `requirements/edx/kernel.in` and re-compile the requirements as a part of this PR.